framework/resources/Datasources/hive_storage/change_metadata2.ddl

```diff
@@ -34,45 +34,45 @@ ALTER TABLE remove_columns2 REPLACE COLUMNS (
 ALTER TABLE modify_columntypes1 REPLACE COLUMNS (
   int_col string,
   bigint_col varchar(50),
-  date_col int,
-  time_col bigint,
+  date_col date,
+  time_col string,
   timestamp_col varchar(20),
-  interval_col date,
-  varchar_col int,
-  float_col int,
-  double_col float,
-  bool_col int
+  interval_col string,
+  varchar_col string,
+  float_col float,
+  double_col double,
+  bool_col boolean
 );
 
 ALTER TABLE modify_columntypes2 REPLACE COLUMNS (
   int_col float,
-  bigint_col int,
+  bigint_col bigint,
   date_col varchar(15),
-  time_col timestamp,
-  timestamp_col date,
-  interval_col boolean,
+  time_col string,
+  timestamp_col timestamp,
+  interval_col string,
```

Comment:
Should interval be string, or long?

Reply:
When the table was created the data type assigned to interval was string, retained the same. 
Table is created in the change_metadata1.ddl

Reply:
cool, thanks! @rchallapalli will be the best person to share more details on the intended behavior.